solo.json.erb

```
@@ -0,0 +1,32 @@
+<%=
+require 'json'
+{
```

Can fail fast using `raise RuntimeError` when required variables are not supplied like in [0]

[0] https://github.com/sonatype/docker-nexus-iq-server/pull/1

---

Added error checks for the three "required" variables.